Documentation/ExtensionInstallation/Index.rst

```diff
@@ -97,39 +108,45 @@ On the left side you see an icon, which shows the status of each extension, and
 Next to the extension you want to deinstall click on Extension UnInstall Icon. After some seconds the icon changes to
 the grey Extension Install Icon.
 
-Uninstall Extension Manually
+.. _install-extension-manually:
+
+Install / Uninstall an Extension Manually
```

Comment:
See above: I'd rather have a decision on whether it's uninstall or deinstall. ("Remove" == "Delete" is the same case for me)

Reply:
well deinstall is clearly denglish when you do a search for it. Remove and Delete are synonyms and both have been used in the past by the TYPO3 Backend. But the Hovertext of the Button in the Extension Manager says "Remove" now.  I just put in the overview once that "Removing an Extensions means deleting it from disc". 
However a google search for "delete TYPO3 Extension" gives you twice as much results as "remove TYPO3 Extension"